### ISSUE TITLE
An error was encountered using the resampling feature.

### ISSUE THREAD

**Issue by 49xxy** [kynkaat/role-of-imagenet-classes-in-fid#2]
Thank you for your excellent work.
I encountered the following problems when using the resampling function. I hope to get your help. 
![image](https://user-images.githubusercontent.com/102204219/199490938-56cc7147-4f91-4261-b4e9-442fc1dda61c.png)
The command I'm running is：
`!python run_resampling.py --zip_path=datasets/face7k.zip --network_pkl=models/network-snapshot-004300.pkl --num_real_images=7120 --feature_mode=logits`

**Comment by kynkaat** [comment]
Hi,

Thank you for reporting this. To reproduce your error, I ran `run_resampling.py` demo command with the FFHQ dataset and released pre-trained generators (5k reals and 25k generated for quicker testing):
```
python run_resampling.py --zip_path=data_path \\
        --network_pkl=https://drive.google.com/uc?id=119HvnQ5nwHl0_vUTEFWQNk4bwYjoXTrC \\
        --feature_mode=logits
 ```
 
And I can't seem reproduce the error. The output I get is following:
```
Reading images and computing features...
Real features shape:         (5000, 1008)
Real metrics features shape: (5000, 2048)
Real CLIP features shape:    (5000, 512)

Generating images and computing features...
Setting up PyTorch plugin "bias_act_plugin"... Done.
Setting up PyTorch plugin "upfirdn2d_plugin"... Done.
Gen. features shape          (25000, 1008)
Gen. metrics features shape: (25000, 2048)
Gen. CLIP features shape:    (25000, 512)

Computing real statistics...
Optimizing FID directly...
Resampled feature shapes: real = (5000, 1008), gen. = (25000, 1008)

It. 1/100000, loss = 25.36533
FID = 11.79
CLIP-FID = 3.38
Elapsed time: 55.34s
...
```

Your error is a some sort of shape mismatch. Looking at your output the real CLIP ViT-B/32 dimension is `N x 1008` and it should be `N x 512` (as in the prints above). Have you accidentally modified the feature array shapes/network names such that they could give you this error?

**Comment by 49xxy** [comment]
Thank you for your reply.
 I have solved the previous problem, which was caused by the excessively high version of my pytorch. I'm hoping to get your help with the following problem in my use of resampling on projected-fastgan：
![image](https://user-images.githubusercontent.com/102204219/200887029-ae1c2697-b375-453a-8b29-462586e0fc24.png)
